### PR TITLE
LPD-41544 | Test Fix | Poshi >>> Playwright 

### PR DESCRIPTION
**Pull request by rafaellajordao**
### Motivation:
Test failing due to Poshi limitations.

### Proposed Solution:
Transcribe test to playwright in order to facilitate maintenance.

**More details:** [LPD-41544](https://liferay.atlassian.net/browse/LPD-41544)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

File: modules/test/playwright/tests/osb-faro-web/assetsWebContent.spec.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+			await page.getByRole('link', {name: 'Web Content Title'}).click();
+
+			expect(page.getByRole('button', {name: 'Views'})).toBeVisible();
+			expect(page.getByText('Asset Appears On')).toBeVisible();
+			expect(
+				page.getByRole('link', {name: 'My Page - My Site'})
+			).toBeVisible();
+			expect(
+				page.getByRole('cell', {
+					name: 'http://localhost:7300/web/my-site',
```
Comment (interaminense) [inline modules/test/playwright/tests/osb-faro-web/assetsWebContent.spec.ts:182]: Can you remove the address to be something like? `/web/my-site`

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:sf

**Pull request closed without merging**